rom/dev/src/start.S

```diff
@@ -342,7 +342,16 @@ _nmi_handler:
 .global exit_rom
 exit_rom:
     .cfi_startproc
-    jr a0
+
+    // Clear the stack
+    // Save the FMC address
+    addi a3, a0, 0
+    la a0, STACK_ORG         // dest
+    la a1, STACK_SIZE        // len 
+    call _zero_mem256
```

Comment:
If we clearing the stack, we should probably also clear all the CPU registers, no?

Reply:
Updated.